### DISCUSSION TITLE
Experiencing high latency

### DISCUSSION THREAD

**Discussion by yarons** [WeblateOrg/weblate#8654] [Q&A] [answered]
Translating Kimai (for example) is really slow, submitting the string and moving on to the next one can sometimes take tens of seconds.

Are you experiencing the same thing with other projects?

**Comment by nijel** [comment, marked as answer]
I can confirm that we see some issues since Friday on Hosted Weblate. Investigating that.

**Reply by nijel** [reply]
We had performance issues with the database server, it should be now resolved.

**Reply by yarons** [reply]
Looks much better now.
